src/controllers/index.ts

```diff
@@ -1,3 +1,3 @@
-import { UserController } from './authController';
+import { registerUser } from './authController';
 
-export{UserController};
\ No newline at end of file
+export{registerUser};
```

Comment:
can you import like this . in `index.ts` export without importing something before. you just like that
![image](https://github.com/atlp-rwanda/knights-ecomm-be/assets/68749686/ab599c5f-0106-4ae3-ac2b-ec2ad257ab16)

Reply:
I think, we should put all our the functions related to the user  in one file for userAuth, It will be better for accessibility thats why am exporting by name . beacuse having a file for every function would make the project cloudy